numba/targets/boxing.py

```diff
@@ -970,6 +970,18 @@ def box_dtype(typ, val, c):
     return c.pyapi.unserialize(c.pyapi.serialize_object(np_dtype))
 
 
+@box(types.ValueDType)
+def box_value_dtype(typ, val, c):
+    np_dtype = numpy_support.as_dtype(typ.dtype)
+    return c.pyapi.unserialize(c.pyapi.serialize_object(np_dtype))
+
+@unbox(types.ValueDType)
+def unbox_value_dtype(typ, obj, c):
```

Comment:
The unboxer can just `return NativeValue(c.context.get_dummy_value())`.  This type do not have any meaningful representation as a value.

Reply:
Glad this was so easy! Done.